[18.0][MIG] purchase_unreconciled: Migration to 18.0

IsabelAForgeFlow opened the pull request:
Migration to 18.0

The changes in the security file were added to avoid the following warning: WARNING odoo odoo.models: Creating record purchase.unreconciled.exceeded.wiz in module purchase_unreconciled.

AaronHForgeFlow committed:
[ADD] purchase_unreconciled

AaronHForgeFlow committed:
[IMP] purchase_unreconciled: black, isort, prettier

AaronHForgeFlow committed:
[15.0][MIG]purchase_unreconciled

AaronHForgeFlow committed:
[MIG] purchase_unreconciled: migration to version 16.0

JasminSForgeFlow committed:
[IMP] purchase_unreconciled: black, isort, prettier

JasminSForgeFlow committed:
[MIG] purchase_unreconciled: Migration to 17.0

Commit:
[UPD] Update purchase_unreconciled.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 90.2% (37 of 41 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-purchase_unreconciled
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-purchase_unreconciled/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (41 of 41 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-purchase_unreconciled
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-purchase_unreconciled/it/

JordiMForgeFlow commented:
@IsabelAForgeFlow dependency has been merged, you can remove the TMP commit :)

IsabelAForgeFlow committed:
[IMP] CI: add rebel module

IsabelAForgeFlow commented:
> @IsabelAForgeFlow dependency has been merged, you can remove the TMP commit :)

Done!

JordiMForgeFlow committed:
[IMP] purchase_unreconciled: adapt to use new field from account_move_line_purchase_info

JordiMForgeFlow commented:
@IsabelAForgeFlow could you add the changes done in this commit: https://github.com/OCA/account-financial-tools/pull/2065/commits/6411ab7b8e144a2866faf3a70d7c2095008ee46b? No need to change the version number in the manifest though :)

IsabelAForgeFlow commented:
> @IsabelAForgeFlow could you add the changes done in this commit: [6411ab7](https://github.com/OCA/account-financial-tools/commit/6411ab7b8e144a2866faf3a70d7c2095008ee46b)? No need to change the version number in the manifest though :)

Done!

AaronHForgeFlow commented on purchase_unreconciled/models/purchase_order.py:
```diff
@@ -0,0 +1,278 @@
+                "name": _("Reconciled journal items"),
+                "type": "ir.actions.act_window",
+                "view_type": "form",
+                "view_mode": "tree,form",
```
```suggestion
                "view_mode": "list,form",
```

Getting this error, I think it is because of this line:

![image](https://github.com/user-attachments/assets/a8b46ebc-0334-4bd4-a7f4-13a9a4fafeb5)

AaronHForgeFlow reviewed (commented):
So far only one issue found

IsabelAForgeFlow committed:
[MIG] purchase_unreconciled: Migration to 18.0

[MIG] purchase_unreconciled: Migration to 18.0

[MIG] purchase_unreconciled: Migration to 18.0

[MIG] purchase_unreconciled: Migration to 18.0

[MIG] purchase_unreconciled: Migration to 18.0

[MIG] purchase_unreconciled: Migration to 18.0

[MIG] purchase_unreconciled: Migration to 18.0

IsabelAForgeFlow commented:
> So far only one issue found

Done!

AaronHForgeFlow reviewed (approved):
Functional review LGTM

AaronHForgeFlow commented:
/ocabot merge nobump
